test/TestAssets/performance/MSTest1Passing/MSTests.tt

```diff
@@ -0,0 +1,52 @@
+﻿using System;
```

Comment:
There is a mix of spaces and tabs, could you save the file from VS to have it fixed please?